### PR TITLE
CosmosDB Account related cmdlets 

### PR DESCRIPTION
**Pull request by MehaKaushik**
## Description

This PR brings in CosmosDB Account related cmdlets
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by wyunchi-ms** [comment]
Hi @MehaKaushik , please check the CI Analysis failed.

File: src/CosmosDB/CosmosDB/CosmosDB.csproj [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>CosmosDB</PsModuleName>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
```
Comment (msJinLei) [inline src/CosmosDB/CosmosDB/CosmosDB.csproj]: Do not specify target framework here.

**Comment by isra-fel** [comment]
Kindly ping @MehaKaushik

**Comment by msJinLei** [comment]
@MehaKaushik  We will close this pr in few days since you have no response for a long time. You can create a new pr when you code is ready.

**Comment by MehaKaushik** [comment]
Apologies for the delayed response, was OOF previously.  Closing this PR,since it uses Fluent SDK. Waiting for the native SDK for CosmosDB to be released.

**Pull request closed without merging**